File: PRAK404-2510817220023-NURFITRIA RAMADHANI.c
#include<stdio.h>

int main(){
int pil=0;
float nil1,nil2,hasil;

while(pil != 5){
printf("Pilih program\n1. Penjumlahan\n2. Pengurangan\n3. Perkalian\n4. Pembagian\n5. Exit\nMasukkan Pilihan : ");
scanf("%d", &pil);
if(pil==1){
    printf("Masukkan nilai pertama : ");
    scanf("%f", &nil1);
    printf("Masukkan nilai kedua : ");
    scanf("%f", &nil2);
    hasil = nil1+nil2;
    printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f",nil1,nil2,hasil);
    printf("\n\n");
}else if(pil==2){
    printf("Masukkan nilai pertama : ");
    scanf("%f", &nil1);
    printf("Masukkan nilai kedua : ");
    scanf("%f", &nil2);
    hasil = nil1-nil2;
    printf("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f",nil1,nil2,hasil);
    printf("\n\n");
}else if(pil==3){
    printf("Masukkan nilai pertama : ");
    scanf("%f", &nil1);
    printf("Masukkan nilai kedua : ");
    scanf("%f", &nil2);
    hasil = nil1*nil2;
    printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f",nil1,nil2,hasil);
    printf("\n\n");
}else if(pil==4){
    printf("Masukkan nilai pertama : ");
    scanf("%f", &nil1);
    printf("Masukkan nilai kedua : ");
    scanf("%f", &nil2);
    hasil = nil1/nil2;
    printf("Hasil pembagian antara %.2f dengan %.2f adalah %.2f",nil1,nil2,hasil);
    printf("\n\n");
}else if(pil==5){
    printf("Terimakasih telah menggunakan Kalkulator Nurfitria Ramadhani");
}else{
    printf("Input anda salah, silakan coba lagi");
    printf("\n\n");
}
}
return 0;
}

File: PRAK404-2510817220023-NURFITRIA RAMADHANI.py
while True:
    pil = int(input("Pilih program\n1. Penjumlahan\n2. Pengurangan\n3. Perkalian\n4. Pembagian\n5. Exit\nMasukkan Pilihan : "))
    if pil==1:
        nil1=float(input("Masukkan nilai pertama : "))
        nil2=float(input("Masukkan nilai kedua : "))
        hasil = nil1+nil2
        print(f"Hasil penjumlahan antara {nil1:.2f} dengan {nil2:.2f} adalah {hasil:.2f}\n")
    elif pil==2:
        nil1=float(input("Masukkan nilai pertama : "))
        nil2=float(input("Masukkan nilai kedua : "))
        hasil = nil1-nil2
        print(f"Hasil pengurangan antara {nil1:.2f} dengan {nil2:.2f} adalah {hasil:.2f}\n")
    elif pil==3:
        nil1=float(input("Masukkan nilai pertama : "))
        nil2=float(input("Masukkan nilai kedua : "))
        hasil = nil1*nil2
        print(f"Hasil perkalian antara {nil1:.2f} dengan {nil2:.2f} adalah {hasil:.2f}\n")
    elif pil==4:
        nil1=float(input("Masukkan nilai pertama : "))
        nil2=float(input("Masukkan nilai kedua : "))
        hasil = nil1/nil2
        print(f"Hasil pembagian antara {nil1:.2f} dengan {nil2:.2f} adalah {hasil:.2f}\n")
    elif pil==5:
        print("Terimakasih telah menggunakan Kalkulator Nurfitria Ramadhani")
        break
    else:
        print("Input anda salah, silakan coba lagi\n")